charts/stable/factorio/values.yaml

```diff
@@ -0,0 +1,371 @@
+#
+# IMPORTANT NOTE
+#
+# This chart inherits from our common library chart. You can check the default values/options here:
+# https://github.com/k8s-at-home/library-charts/tree/main/charts/stable/common/values.yaml
+#
+
+image:
+  # -- image repository
+  repository: factoriotools/factorio
+  # -- image tag
+  # @default -- chart.appVersion
+  tag:
+  # -- image pull policy
+  pullPolicy: IfNotPresent
+
+initContainers:
+  # -- Configures an initContainer that copies the configmap to the factorio conf directory
+  # @default -- See values.yaml
+  copy-configmap:
+    image: busybox
+    imagePullPolicy: IfNotPresent
+    command:
+      - "sh"
+      - "-c"
+      - |
+        mkdir -p /factorio/config
+        cp -vL /helmconfig/* /factorio/config/
+    volumeMounts:
+      - name: factorio-config
+        mountPath: /helmconfig/
+      - name: config
+        mountPath: /factorio/
+    securityContext:
+      runAsUser: 0
+
+# -- environment variables. See more environment variables in the [factorio documentation](https://factorio.org/docs).
+# @default -- See below
+env:
+  # -- Set the container timezone
+  TZ: UTC
+  # -- Generate a new save if one does not exist before starting the server
+  GENERATE_NEW_SAVE: "{{ .Values.config.factorioServer.generate_new_save }}"
+  # -- Load latest when true. Otherwise load SAVE_NAME
+  LOAD_LATEST_SAVE: "{{ .Values.config.factorioServer.load_latest_save }}"
+  # -- UDP port the server listens on
+  PORT: "{{ .Values.service.main.ports.server.port }}"
+  # -- TCP port the rcon server listens on
+  RCON_PORT: "{{ .Values.service.main.ports.rcon.port }}"
+  # -- Name to use for the save file
+  SAVE_NAME: "{{ .Values.config.factorioServer.save_name }}"
+  # -- factorio.com token
+  TOKEN: "{{ .Values.config.factorioServer.token }}"
+  # -- If mods should be updated befoqre starting the server
+  UPDATE_MODS_ON_START: "{{ .Values.config.factorioServer.update_mods_on_start }}"
+  # -- factorio.com username
+  USERNAME: "{{ .Values.config.factorioServer.username }}"
+
+# -- Configures service settings for the chart.
+# @default -- See values.yaml
+service:
+  main:
+    ports:
+      http:
+        enabled: false
+        primary: false
+      server:
+        enabled: true
+        port: 34197
+        protocol: UDP
+      rcon:
+        enabled: true
+        primary: true
+        port: 27015
+        protocol: TCP
+
+ingress:
+  # -- Enable and configure ingress settings for the chart under this key.
+  # @default -- See values.yaml
+  main:
+    enabled: false
+
+# -- Configure persistence settings for the chart under this key.
+# @default -- See values.yaml
+persistence:
+  config:
+    enabled: true
+    mountPath: /factorio
+# -- Configure server settings.  Can be disabled if you wish to configure in the container in the json config files.
```

Comment:
```suggestion
    mountPath: /factorio

# -- Configure server settings.  Can be disabled if you wish to configure in the container in the json config files.
```
Also maybe add a reference here to somewhere that explains these settings?

Reply:
Done - pushed another commit to this branch.  Unfortunately not a good place to find all the setting descriptions online however, the official docs assume the var name is enough in some spots :(